tests/etl/test_perf_data_adapters.py

```diff
@@ -50,4 +51,22 @@ def test_adapt_and_load():
         datum['blob'] = json.dumps({'talos_data': [datum['blob']]})
         tda.adapt_and_load(reference_data, job_data, datum)
 
+        # we upload a summary with a suite and subtest values, +1 for suite
+        if 'summary' in datum['blob']:
```

Comment:
I see, I guess this is o.k. I would at least add an:

    else:
         # FIXME: the talos data blob we're currently using contains datums with summaries and those without
         # we should probably test non-summarized data as well
         pass

after the if clause to make it more clear what's happening